build.psm1

```diff
@@ -1821,20 +1822,22 @@ function Install-Dotnet {
     if ($environment.IsLinux -or $environment.IsMacOS) {
         $wget = Get-Command -Name wget -CommandType Application -TotalCount 1 -ErrorAction Stop
 
-        # Uninstall all previous dotnet packages
-        $uninstallScript = if ($environment.IsLinux -and $environment.IsUbuntu) {
-            "dotnet-uninstall-debian-packages.sh"
-        } elseif ($environment.IsMacOS) {
-            "dotnet-uninstall-pkgs.sh"
-        }
+        # Attempt to uninstall previous dotnet packages if requested
+        if ($RemovePreviousVersion) {
```

Comment:
Shouldn't `$RemovePreviousVersion` also be checked for non-Windows platforms?

Reply:
.NET doesn't provide that ability